index.html

```diff
@@ -20,10 +20,306 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <!--
+    <!--Content within a section are all in their own divs so that I can align them according to their personal boxes-->
+    <!--Added the a tags on the buttons by mistake, but now when I try to get rid of them, it messes up the alignment. Not sure how to fix that-->
 
-      YOU HAVE TO MODIFY THE HTML TO BUILD YOUR AWESOME PAGE
+    <!--Navigation Menu-->
+    <header>
+      <div class="header">
+        <div class="container">
+          <div class="navigation">
+            <h2 class="sr-only">Site Header</h2>
 
-    -->
-  </body>
-</html>
+            <!--Logo-->
+            <img
+              class="header--navigation--logo"
+              src="./images/Wensight-Logo.png"
+              height="80"
+              alt="Company Logo"
+              />
+
+              <!--Search Bar-->
+            <form class="search--bar" >
+              <input
+              name="search--products" 
+              id="search--products"
+              type="text" 
+              placeholder="Search Products">
+              <button type="submit">
+                <img 
+                src="./images/Search-icon.png"
+                alt="Search button icon"
+                height="50">
+              </button>
+            </form>
+           
+            <!--Navigation Menu items-->
+            <nav class="header--navigation--list">
+              <ul>
+                <li>
+                  <a href="">News</a>
+                </li>
+                <li>
+                  <a href="">Watch</a>
+                </li>
+                <li>
+                  <a href="">Shop</a>
+                </li>
+                <li>
+                  <a href="">Quizzes and Features</a>
+                </li>
+                <li>
+                  <a href="">Help</a>
+                </li>
+              </ul>
+            </nav>
+          </div>
+        </div>
+      </div>
+    </header>
+    <!--End of Navigation Menu-->
+
+    <!--Start of main cite content-->
+      <main>
+        <!--Hero Section-->
+        <section>
+          <div class="Hero--Section">
+            <div class="Hero--text">
+              <h2 class="sr-only">Hero Section</h2>
```

Comment:
This is honestly excellent work @naomileron 

The only thing that you are missing is an `<h1>` in your `<main>` area. 

You can hide it visually using CSS (https://tailwindcss.com/docs/screen-readers)